modules/simple_forms_api/app/services/simple_forms_api/metadata_validator.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class MetadataValidator
+    def self.validate(metadata)
+      validate_first_name(metadata)
+        .then { |m| validate_last_name(m) }
+        .then { |m| validate_file_number(m) }
+        .then { |m| validate_zip_code(m) }
+        .then { |m| validate_source(m) }
+        .then { |m| validate_doc_type(m) }
+    end
+
+    def self.validate_first_name(metadata)
+      validate_presence_and_stringiness(metadata['veteranFirstName'], 'veteran first name')
+      metadata['veteranFirstName'] = metadata['veteranFirstName'][0..49]
```

Comment:
unfamiliar with ruby syntax sometimes.
Is the goal to validate the name is less than 50 characters? or to drop any characters more than 50?